src/app/features/tech-record/components/tech-record-title/tech-record-title.component.html

```diff
@@ -36,6 +36,16 @@ <h1 class="govuk-heading-l">
       <dd id="current-vrm" class="govuk-summary-list__value">
         <app-number-plate *ngIf="currentVrm" [vrm]="currentVrm"></app-number-plate>
       </dd>
+      <dd class="govuk-summary-list__actions"></dd>
+      <dd class="govuk-summary-list__actions">
+        <app-button
```

Comment:
I did some changes to this component last week to align the actions in columns. You might not have the latest code so can you pull it down and follow the example.